Formula/pocsuite3.rb

```diff
@@ -0,0 +1,143 @@
+class Pocsuite3 < Formula
+  include Language::Python::Virtualenv
+
+  desc "Open-sourced remote vulnerability testing framework"
+  homepage "https://pocsuite.org/"
+  url "https://github.com/knownsec/pocsuite3/archive/v1.8.7/pocsuite3-1.8.7.tar.gz"
+  sha256 "2374ee3172ef566d2f5ded9327abe8ed292563bd11ce623cf54bf2a133996a71"
+  license "GPL-2.0-only"
+  head "https://github.com/knownsec/pocsuite3.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Is this needed? You can check by removing the `livecheck` block and inspecting the output of `brew livecheck --debug pocsuite3`.

We try to avoid `:github_latest` because this gets rate-limited. Usually the default `livecheck` behaviour for GitHub URLs is good enough.

Reply:
I have removed it, Thanks for the review.